.github/workflows/gempush.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         touch $HOME/.gem/credentials
         chmod 0600 $HOME/.gem/credentials
         printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
-        gem build *.gemspec
-        gem push *.gem
+        bin/rake logfmt:release
```

Comment:
If we like the separate gems approach, I'll add `bin/rake logfmt-tagged_logger:release` here too.

Reply:
@cyberdelia I think to enable automatically pushing the `logfmt-tagged_logger` gem you'l need to accept the invite on RubyGems since it uses your creds.